demo/agent_chatbot/utils.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
In my local environment, I'm seeing some type issues that CI isn't catching. I've added this type ignore since they only relate to a few demos. Will revisit later.